tests/backup/backup_helper.go

```diff
@@ -1187,7 +1194,7 @@ func backupSuccessCheck(backupName string, orgID string, retryDuration int, retr
 	if err != nil {
 		return false, err
 	}
-	backupStatus := (resp.GetBackup().GetStatus().Status == api.BackupInfo_StatusInfo_Success)
+	backupStatus := resp.GetBackup().GetStatus().Status == api.BackupInfo_StatusInfo_Success
```

Comment:
Will this status always be equal to Success here? Why even compare it here if we don't check it here? Could just return `true`

Reply:
Here we are not equating the backupStatus to Success.
We are cheking if resp.GetBackup().GetStatus().Status is equal to api.BackupInfo_StatusInfo_Success or not(which will be either true or false.
This true or false will be initialised to backupStatus and hence we are returning a bool value

Reply:
I removed the parenthesis as it was showing redundant parenthesis error. Should I not remove the parenthesis?

Reply:
Understood now.. Corrected